src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import settings from '../../globals/js/settings';
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import InitComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import handles from '../../globals/js/mixins/handles';
+
+class PaginationNav extends mixin(createComponent, InitComponentBySearch, handles) {
+  /**
+   * Pagination Nav component
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @extends Handles
+   * @param {HTMLElement} element The element working as a pagination nav.
+   */
+  constructor(element, options) {
+    super(element, options);
+    this.element.addEventListener('click', evt => this.handleClick(evt));
+    this.selectorPageElementArray = this.element.querySelectorAll(this.options.selectorPageElement);
+    this.totalPages = this.selectorPageElementArray.length;
+    this.currentIndex = 0;
```

Comment:
I'd suggest not maintaining current index as an instance property, considering dynamic DOM updates. Instead, I'd suggest creating a method to determine the current index.